check_config_win.py

```diff
@@ -0,0 +1,64 @@
+'''
```

Comment:
Add a shebang here: `#!/usr/bin/env python3`. I know this doesn't matter terribly on Windows, but it helps editors pick up on the intended syntax and might be nice for Cygwin/WSL (?) users.